src/app/core-ui/main/announce-notification/announcement-notification.component.ts

```diff
@@ -33,7 +38,7 @@ export class ReleaseNotificationComponent implements OnInit, OnDestroy {
 
   getCurrentClientVersion() {
```

Comment:
It is probably not necessary for this method to exist. The component exists here to simply inform users of notifications. Thus its not necessary to be checking for new version releases in this component.